src/app/Models/Address.php

```diff
@@ -52,6 +52,8 @@ public function setDefault()
                 ->whereIsDefault(true)
                 ->update(['is_default' => false]);
 
+            $this->refresh();
```

Comment:
we don't really need this because the interface does not allow accessing this route when the address is already default. I'll add an extra verification in the `DefaultAddress` controller to avoid this.